client/src/transmitter/index.ts

```diff
@@ -1,6 +1,7 @@
 import isIp from 'is-ip'
 
 import { copyToClipboard } from '../clipboard'
+import cast from './cast'
 
 const isLocalHost = (hostname) => {
```

Comment:
`isLocalHost` should probably become a module.

Reply:
yeah there is a couple of location-parsing/client-url-generating code that is repeated now, but I think it's not complex or core enough to need to have a module for it. I actually think we may be able to get rid of most of it, as you also noticed some possible conditions that don't seem to make much sense. 

IMO we should probably refactor this later, so I'll skip it for now.